### DISCUSSION TITLE
add right-to-left (RTL) style support for RTL languages like Persian and Arabic , ...

### DISCUSSION THREAD

**Discussion by dnyall** [umputun/remark42#1832] [Ideas]
I want to thank you in advance for your excellent project.

I think it would be better to support right-to-left languages by using direction: rtl or text-align: rtl."

**Comment by paskal** [comment]
It was improved in https://github.com/umputun/remark42/pull/1799 and already published as v1.14.0, is there anything which doesn't work for you? Can you reproduce the problem on https://remark42.com/demo/ ?

**Reply by dnyall** [reply]
No, I didn't mean the text-align of the field.
I mean the placement of elements and buttons base on RTL direction.

**Reply by paskal** [reply]
Can you please show an interface which is altering like that, which has LTR and RTL versions? Any example or a few if you have them would be useful.
